<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherLogsPage</class>
 <widget class="QWidget" name="OtherLogsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>538</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="2">
    <widget class="QPushButton" name="findButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Find</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="btnBottom">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Scroll all the way to bottom</string>
     </property>
     <property name="text">
      <string>&amp;Bottom</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="LogView" name="text">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string notr="true"/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="5">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="selectLogBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDelete">
         <property name="toolTip">
          <string>Delete the selected log</string>
         </property>
         <property name="text">
          <string>&amp;Delete Selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClean">
         <property name="toolTip">
          <string>Delete all the logs</string>
         </property>
         <property name="text">
          <string>Delete &amp;All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="5">
      <layout class="QHBoxLayout" name="horizontalLayout1">
       <item>
        <widget class="QCheckBox" name="trackLogCheckbox">
         <property name="text">
          <string>Keep updating</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="wrapCheckbox">
         <property name="text">
          <string>Wrap lines</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="colorCheckbox">
         <property name="text">
          <string>Color lines</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCopy">
         <property name="toolTip">
          <string>Copy the whole log into the clipboard</string>
         </property>
         <property name="text">
          <string>&amp;Copy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPaste">
         <property name="toolTip">
          <string>Upload the log to the paste service configured in preferences</string>
         </property>
         <property name="text">
          <string>&amp;Upload</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReload">
         <property name="toolTip">
          <string>Reload the contents of the log from the disk</string>
         </property>
         <property name="text">
          <string>&amp;Reload</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="searchBar">
     <property name="placeholderText">
      <string>Search</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogView</class>
   <extends>QPlainTextEdit</extends>
   <header>ui/widgets/LogView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>selectLogBox</tabstop>
  <tabstop>btnReload</tabstop>
  <tabstop>btnCopy</tabstop>
  <tabstop>btnPaste</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnClean</tabstop>
  <tabstop>wrapCheckbox</tabstop>
  <tabstop>colorCheckbox</tabstop>
  <tabstop>text</tabstop>
  <tabstop>searchBar</tabstop>
  <tabstop>findButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
